src/main/java/at/hannibal2/skyhanni/features/garden/farming/CropMoneyDisplay.kt

```diff
@@ -126,7 +126,7 @@ class CropMoneyDisplay {
             number++
             val cropName = cropNames[internalName]!!
             val isCurrent = cropName == GardenAPI.cropInHand
-            if (number > config.moneyPerHourShowOnlyBest && !isCurrent) continue
+            if (number > config.moneyPerHourShowOnlyBest && (!config.moneyPerHourShowCurrent && isCurrent || !isCurrent)) continue
```

Comment:
I'm pretty sure this equivalent (and also less confusing with operator precedence):

```suggestion
            if (number > config.moneyPerHourShowOnlyBest && (!config.moneyPerHourShowCurrent || !isCurrent)) continue
```

Reply:
you are right, forgot to delete it because I did it only for me first and didn't care. thanks